packages/core/src/Stores/client-store.js

```diff
@@ -2505,8 +2505,12 @@ export default class ClientStore extends BaseStore {
     get is_high_risk() {
         if (isEmptyObject(this.account_status)) return false;
         const { gaming_company, financial_company } = this.landing_companies;
+
+        //this is a conditional check for countries like Australia/Norway which fulfiles one of these following conditions
+        const restricted_countries = financial_company?.shortcode === 'svg' || gaming_company?.shortcode === 'svg';
+
```

Comment:
@sergei-deriv  For a low risk CR client, we have the rule -
 ```
  financial_company?.shortcode === 'maltainvest' &&
            gaming_company?.shortcode === 'svg' 
```
 
 So if we are adding a condition like "**financial_company?.shortcode === 'svg' || gaming_company?.shortcode === 'svg'**` on this condition , those clients will also become restrictred countries. isnt? Can u double check?

Reply:
@amina-deriv If we are adding this condition for low_risk for Australia and Norway, yes, we will have 'true', but if I understand it right this condition related only for high risk. As you wrote me about 'content_flag' in 'traders-hub-store', this condition uses for high_risk only.

Reply:
@sergei-deriv @amina-deriv is right.
```suggestion
financial_company?.shortcode === 'svg' || (gaming_company?.shortcode === 'svg' && financial_company?.shortcode !== 'maltainvest')`
```

Reply:
@mahdiyeh-deriv I thought @amina-deriv suggested this `(financial_company?.shortcode === 'svg' || gaming_company?.shortcode === 'svg') || (gaming_company?.shortcode === 'svg' && financial_company?.shortcode !== 'maltainvest')`

Reply:
@sergei-deriv In content flag, The condition to check low risk CR had higher priority, so it returned expected values. 
<img width="960" alt="Screenshot 2023-02-20 at 11 52 39 AM" src="https://user-images.githubusercontent.com/84661147/220045082-a68fe7f5-a6b0-462e-8b5d-ff910ee766e2.png">

Reply:
> @mahdiyeh-deriv I thought @amina-deriv suggested this `(financial_company?.shortcode === 'svg' || gaming_company?.shortcode === 'svg') || (gaming_company?.shortcode === 'svg' && financial_company?.shortcode !== 'maltainvest')`

@sergei-deriv i dont think this will work. Can we try the way @mahdiyeh-deriv suggested?